consensus/polybft/consensus_runtime.go

```diff
@@ -46,8 +46,6 @@ type txPoolInterface interface {
 type checkpointBackend interface {
 	// BuildEventRoot generates an event root hash from exit events in given epoch
 	BuildEventRoot(epoch uint64) (types.Hash, error)
-	// InsertExitEvents inserts provided exit events to persistence storage
-	InsertExitEvents(exitEvents []*ExitEvent) error
 }
```

Comment:
If this becomes part of checkpointManager, then move this interface to `checkpoint_manager.go` as well.

Reply:
Then we won't need this interface at all.